dvf.libraries.yml

```diff
@@ -19,7 +19,7 @@ d3:
     url: https://github.com/d3/d3/blob/master/LICENSE
     gpl-compatible: true
   js:
-    https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.min.js: { type: external, minified: true }
```

Comment:
Any reason why you didn't use cdnjs? https://cdnjs.cloudflare.com/ajax/libs/d3/5.5.0/d3.min.js

Reply:
certainly preferable from a content security policy PoV?

Reply:
(sidenote - I'd love to see https://www.drupal.org/project/drupal/issues/2927356 land!)

Reply:
Sorry at the time Cloudflare CDN wasn't listing the latest version of D3. It seems to have updated in the meantime, so should be fixed now.

Reply:
@rob-duplock no dramas. Good thing I've checked though :-)